pythonforandroid/recipe.py

```diff
@@ -23,6 +24,12 @@
 from pythonforandroid.util import load_source as import_recipe
 
 
+# jqueryui.com returns a 403 w/ the default user agent
+url_opener = urllib.request.build_opener()
+url_opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+urllib.request.install_opener(url_opener)
```

Comment:
Doesn't this change it at global level? Can't we use some kind of context manager or something so it's rolled back after the recipe download is done?

Reply:
It does. But since this is the only use of urllib in p4a and it seems unlikely that a different user agent would matter to any other code anyway, it seemed like the easiest solution.

Reply:
I changed the code. Is this better?

Reply:
Thanks, looks good to me